### ISSUE TITLE
Use Engines without common URL

### ISSUE THREAD

**Issue by Mike-Reddick** [CanisLupus/swift-selection-search#174]
Some services haven't an "Search URL" for instance "https://www.qualoperadora.net/" when you search the URL don't change, I think can be good add an feature to capture the search engine, using an context menu entry, like the extension "https://addons.mozilla.org/en-US/firefox/addon/burning-moth-add-search/", but to add direct to swift-selection-search

thanks so much!

**Comment by CanisLupus** [comment]
Yup, these are POST-based search engines, so they don't use the URL and instead make a different request to the server. Noted down. ;) Cheers

**Comment by Mike-Reddick** [comment]
thanks so much! no problem.

**Comment by BubiBalboa** [comment]
Is there a way to add POST searches to SSS? I've been searching around and couldn't find anything about that.

**Comment by CanisLupus** [comment]
Hi @BubiBalboa, sorry, unfortunately no! Currently SSS only supports engines that use an actual URL. POST-based engines would be great, though. I'd like to do that at some point.
